tests/suite/resources_utils.py

```diff
@@ -750,6 +782,21 @@ def get_ingress_nginx_template_conf(v1: CoreV1Api, ingress_namespace, ingress_na
     return get_file_contents(v1, file_path, pod_name, pod_namespace)
 
 
+def get_ts_nginx_template_conf(v1: CoreV1Api, resource_namespace, resource_name, ingress_name, ingress_namespace) -> str:
+    """
+    Get contents of /etc/nginx/stream-conf.d/ts_{namespace}-{resource_name}.conf in the pod.
+
+    :param v1: CoreV1Api
+    :param resource_namespace:
+    :param resource_name:
+    :param ingress_name:
+    :param ingress_namespace:
+    :return: str
+    """
+    file_path = f"/etc/nginx/stream-conf.d/ts_{resource_namespace}_{resource_name}.conf"
+    return get_file_contents(v1, file_path, ingress_name, ingress_namespace)
```

Comment:
just noticed, ingress_name and ingress_namespace are actually IC pod_name and pod_namespace. Could you rename them?
```
def get_file_contents(v1: CoreV1Api, file_path, pod_name, pod_namespace) -> str:
```

Reply:
fixed